engine/src/main/java/org/terasology/world/block/items/OnBlockItemPlaced.java

```diff
@@ -24,17 +24,27 @@
 public class OnBlockItemPlaced implements Event {
     private Vector3i position;
     private EntityRef placedBlock;
+    private EntityRef instigator;
```

Comment:
Ideally this would be set to `EntityRef.NULL` rather than `null`
That can either be done here or by calling the three arg constructor from the two arg constrctor